web/views/note_view.ex

```diff
@@ -1,4 +1,12 @@
 defmodule Notox.NoteView do
   use Notox.Web, :view
 
+  defp sanitize_html(unsafe) do
+    {:safe, HtmlSanitizeEx.html5(unsafe)}
```

Comment:
Also, any reason you moved away from `raw`? Wrapping it in `{:safe, _}` is essentially doing what `raw` does https://hexdocs.pm/phoenix_html/Phoenix.HTML.html#raw/1